### PR TITLE
MicroProfile 6.0 RC1 Update

### PR DESCRIPTION
**Pull request by jclingan**
This version is in draft form.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -94,7 +94,7 @@ link:https://github.com/eclipse/microprofile-fault-tolerance[MicroProfile Fault
 link:https://github.com/eclipse/microprofile-rest-client[MicroProfile Rest Client] -- MicroProfile type-safe REST client +
 link:https://github.com/eclipse/microprofile-open-api[MicroProfile OpenAPI] -- MicroProfile OpenAPI and annotations +
 link:https://github.com/eclipse/microprofile-jwt-auth[MicroProfile JWT Authentication] -- MicroProfile JWT authentication propagation +
-link:https://github.com/eclipse/microprofile-opentracing[MicroProfile OpenTracing] -- MicroProfile OpenTracing integration +
+link:https://github.com/eclipse/microprofile-telemetry[MicroProfile Telemetry] -- MicroProfile OpenTracing integration +
```
Comment (Emily-Jiang) [inline README.adoc:97]:
```suggestion
link:https://github.com/eclipse/microprofile-telemetry[MicroProfile Telemetry] -- MicroProfile Telemetry integration +
```

File: pom.xml [hunk 2]
```diff
@@ -104,21 +104,17 @@
...
         <!-- MicroProfile specs  -->
-        <config-version>3.0</config-version>
-        <ft-version>4.0</ft-version>
-        <health-version>4.0</health-version>
-        <metrics-version>4.0</metrics-version>
+        <config-version>3.0.2</config-version>
+        <ft-version>4.0.1</ft-version>
+        <health-version>4.0.1</health-version>
+        <metrics-version>5.0.0-RC3</metrics-version> <!-- TODO Update to 5.0 -->
         <jwt-version>2.0</jwt-version>
```
Comment (Emily-Jiang) [inline pom.xml:114]:
```suggestion
        <jwt-version>2.1-RC5</jwt-version>
```

File: pom.xml [hunk 3]
```diff
@@ -237,34 +225,21 @@
                 <artifactId>microprofile-rest-client-api</artifactId>
                 <version>${rest-client-version}</version>
             </dependency>
+<!-- TODO I'm assuming we don't need this since telemetry does not have an MP API
```
Comment (Emily-Jiang) [inline pom.xml:228]: yes

File: pom.xml [hunk 4]
```diff
@@ -294,10 +269,12 @@
             <groupId>org.eclipse.microprofile.rest.client</groupId>
             <artifactId>microprofile-rest-client-api</artifactId>
         </dependency>
+<!-- TODO I'm assuming we don't need this since telemetry does not have an MP API
```
Comment (Emily-Jiang) [inline pom.xml:272]: yes

Comment (jclingan) [reply pom.xml:272]: Yes "we need it"? or "yes, it can be removed"?

Comment (Emily-Jiang) [reply pom.xml:272]: yes, it can be removed.

File: pom.xml [hunk 5]
```diff
@@ -294,10 +269,12 @@
...
         <dependency>
-            <groupId>org.eclipse.microprofile.opentracing</groupId>
-            <artifactId>microprofile-opentracing-api</artifactId>
+            <groupId>org.eclipse.microprofile.telemetry.tracing</groupId>
```
Comment (Emily-Jiang) [inline pom.xml:274]: We don't need this line and the next line.

File: spec/src/main/asciidoc/required-apis.asciidoc [hunk 6]
```diff
@@ -23,37 +23,43 @@
...
 
 These APIs and versions guarantee application developers a minimum set of APIs, but they are not intended to be
 restrictive.
 
 [[javase]]
 === Java SE
 
-Libraries in use by the MicroProfile leverage new features provided in Java SE 8, therefore Java SE 7 and before are
-unsupported.
+MicroProfile APIs and specifications use Java SE 11 for both source and target compatibility. MicroProfile
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/required-apis.asciidoc:46]: This is not true. Only MicroProfile Metrics 5.0 and MicroProfile Telemetry uses Java SE 11 while the others are still on Java SE 8.

File: spec/src/main/asciidoc/required-apis.asciidoc [hunk 7]
```diff
@@ -62,50 +68,21 @@ Use of implementations beyond CDI 3.0 in MicroProfile is allowed, however, not r
  - http://cdi-spec.org/
 
 [[jakarta-annotations]]
-=== Jakarta Annotations 2.0
+=== Jakarta Annotations 2.0 (TODO 2.1?)
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/required-apis.asciidoc:71]: We should remove this section as well as we should just simply depend on Jakarta EE 10 Core Profile.

File: spec/src/main/asciidoc/required-apis.asciidoc [hunk 8]
```diff
@@ -124,45 +101,50 @@ MicroProfile Health provides the ability to expose the health of an application
...
+ - https://github.com/eclipse/microprofile-open-api/releases/tag/3.1
+
+[[mp-telemetry]]
+=== MicroProfile Telemetry 1.0
+
+MicroProfile Telemetry
+Telemetry data is needed to power observability products.
 
-[[mp-opentracing]]
-=== MicroProfile OpenTracing 3.0
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/required-apis.asciidoc:155]: This section should be removed. However, at the end of the section of MicroProfile Telemetry 1.0, we should state soemthing like : MicroProfile Telemetry 1.0 supercedes MicroProfile OpenTracing due to the merge of OpenTracing and OpenSensus into OpenTelemetry. As a result, MicroProfile 6.0 will not include MicroProfile OpenTracing, which will become a standalone specification. It is highly recommended to move away from MicroProfile OpenTracing and use MicroProfile Telemetry instead.

**Review by Emily-Jiang — APPROVED** [review]
This is good for the first try and then reiterate with the polish up.

**Pull request merged**